.github/CONTRIBUTING.md

```diff
@@ -480,8 +480,6 @@ The `SHOULD_CALL_PARENT(TRUE)` lint should be added to ensure that overrides/chi
 ### Use descriptive and obvious names
 Optimize for readability, not writability. While it is certainly easier to write `M` than `victim`, it will cause issues down the line for other developers to figure out what exactly your code is doing, even if you think the variable's purpose is obvious.
 
-#### Don't use abbreviations
-Avoid variables like C, M, and H. Prefer names like "user", "victim", "weapon", etc.
```

Comment:
This can stil be used as an example if it's merged somewhere in the above section